buildscripts/condarecipe.local/meta.yaml

```diff
@@ -42,7 +42,8 @@ requirements:
     - tbb-devel 2021.*       # [not (armv6l or armv7l or aarch64 or linux32 or ppc64le)]
   run:
     - python >=3.7
-    - numpy >=1.18,<1.23
+    # NumPy 1.22.0, 1.22.1, 1.22.2 are all broken for ufuncs, see #7756
+    - numpy >=1.18,<1.23, !=1.22.0, !=1.22.1, !=1.22.2
```

Comment:
The condition in `run_constrained` of ` - numpy !=1.22.0, !=1.22.1, !=1.22.2` is still present?

Reply:
I can confirm this, the entry for `run_constrained` is still present on the HEAD of #8083 cf53c138e51d56da9ecf970abe23417e8779e302